src/submodules/io/jld2_integration.jl

```diff
@@ -75,6 +75,7 @@ struct OSMAgentPositionData
     id::Int
     pos::NTuple{2,Float64}
     dest::NTuple{2,Float64}
+    route::Vector{NTuple{2,Float64}}
```

Comment:
Is this necessary? Why can we not just re-recreate it on-load?

Reply:
Can it be determined whether the route was :shortest or :fastest? Or if it was a return route? If so, it's not necessary.

Reply:
Also, for `:fastest`, the speeds would need to be provided

Reply:
Ah, yes - I see. That needs to be added as a property somewhere probably.

Worry that I have is that the indexes in the route are not transferable due to the OSMX structure, so even adding lat/lon here it means that we'd need to make an additional bunch of methods to re-embed some synthetic route. 

Without the route type property stored that'll be a problem in general, so may make sense to consider that extension somehow. 

Bah, but this could be a per-agent property.... A bit to think about here. 

Feel free to go ahead in the direction you're taking here and we'll see if you hit a wall or not, then we can figure out if there's a simpler design.